service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/OrganizationMapper.java

```diff
@@ -28,11 +33,25 @@
 
     private final NodeUtil nodeUtil;
 
+    public Organization mapOrganization(POCDMT000002UK01InformationRecipient informationRecipient) {
+        POCDMT000002UK01IntendedRecipient intendedRecipient = informationRecipient.getIntendedRecipient();
+        if (intendedRecipient.isSetReceivedOrganization()) {
+            Organization fhirOrganization = mapOrganization(intendedRecipient.getReceivedOrganization());
+            Coding code = new Coding()
+                .setCode(String.valueOf(informationRecipient.getTypeCode()))
+                .setDisplay(nodeUtil.getAllText(intendedRecipient.getReceivedOrganization().getNameArray(0).getDomNode()));
```

Comment:
I was thinking that nodeUtil.getNodeValueString method could be used here (like in the example in my previous comment): `.setDisplay(nodeUtil.getNodeValueString(intendedRecipient.getReceivedOrganization().getNameArray(0)));`